.github/workflows/nightly.yml

```diff
@@ -42,19 +42,19 @@ jobs:
         secrets: inherit
 
     # multi-gpu
-    # AWS-AVX2-192G-4-A10G-96G-Benchmark:
-    #     uses: ./.github/workflows/nm-benchmark.yml
-    #     with:
-    #         label: aws-avx2-192G-4-a10g-96G
-    #         benchmark_config_list_file:  ./.github/data/nm_benchmark_nightly_configs_list.txt
-    #         timeout: 480
-    #         gitref: '${{ github.ref }}'
-    #         Gi_per_thread: 4
-    #         nvcc_threads: 8
-    #         python: "3.10.12"
-    #         # Always push if it is a scheduled job
-    #         push_benchmark_results_to_gh_pages: "${{ github.event_name == 'schedule' || inputs.push_benchmark_results_to_gh_pages }}"
-    #     secrets: inherit
+    AWS-AVX2-192G-4-A10G-96G-Benchmark:
+        uses: ./.github/workflows/nm-benchmark.yml
+        with:
+            label: aws-avx2-192G-4-a10g-96G
+            benchmark_config_list_file:  ./.github/data/nm_benchmark_nightly_configs_list.txt
+            timeout: 480
+            gitref: '${{ github.ref }}'
+            Gi_per_thread: 4
+            nvcc_threads: 8
+            python: "3.10.12"
+            # Always push if it is a scheduled job
+            push_benchmark_results_to_gh_pages: "${{ github.event_name == 'schedule' || inputs.push_benchmark_results_to_gh_pages }}"
+        secrets: inherit
```

Comment:
We turned off the multi-gpu benchmarks intentionally in this PR https://github.com/neuralmagic/nm-vllm/pull/130 - we determined that they are not very useful.

Reply:
okay, i'll delete this section. it is easy enough to add it back.